cli/src/main/kotlin/com/bazel_diff/cli/VersionProvider.kt

```diff
@@ -1,9 +1,16 @@
 package com.bazel_diff.cli
 
 import picocli.CommandLine.IVersionProvider
+import java.io.BufferedReader
+import java.io.InputStreamReader
 
 class VersionProvider : IVersionProvider {
     override fun getVersion(): Array<String> {
-        return arrayOf("7.0.0")
+        val classLoader = this::class.java.classLoader
```

Comment:
Does it make more sense to simply make a bazel rule that can accept the stable-status path and code gen this entire file? Seems like there are a couple steps here that could be collapsed into one?